src/Twig/RecordExtension.php

```diff
@@ -74,10 +74,20 @@ public function dummy_with_env(Environment $env, $input = null)
         return $input;
     }
 
-    public function pager(Environment $env, string $template = '')
+    public function pager(Environment $env, $records = null, string $template = '_sub_pager.twig', ?string $class = null, ?string $theme = null, ?int $surround = null)
     {
-        // @todo See Github issue https://github.com/bolt/four/issues/254
-        return '[pager placeholder]';
+        if (! is_iterable($records)) {
```

Comment:
or just type $records to be an iterable and remove default null?

Reply:
Lemme check if the `Pagerfanta` is accepted as `iterable`. If so, i'll change!